src/model/application/component.rs

```diff
@@ -153,7 +153,7 @@ pub struct SelectMenu {
     #[serde(default)]
     pub options: Vec<SelectMenuOption>,
     /// The result location for modals
-    pub values: Vec<String>,
+    pub values: Option<Vec<String>>,
```

Comment:
This is good, but would you mind changing the type back to `Vec<String>` and adding `#[serde(default)]`? This change constitutes as breaking, which isn't typically allowed on `current` (in certain cases it is, but here it's more viable to fix it temporarily and then add the real fix on `next`; that is, using an `Option`).
```suggestion
    #[serde(default)]
    pub values: Vec<String>,
```